src/App.scss

```diff
@@ -64,7 +64,7 @@ input {
 
 // Removes the scrollbar flash on first load
```

Comment:
Is this comment still valid with the switch to `overflow: hidden` to `overflow: auto`? (as in, scrollbars _may_ now appear)

Reply:
I did not spot any issue with the scrollbar - after the change. Not sure where it was flashing or causing some issues.